apps/poker-roll/src/app/homePage.jsx

```diff
@@ -25,8 +26,8 @@ const Home = (props) => {
                 <div>
                     <h1>Poker Roll</h1>
                 </div>
-                <div>
-                    <SessionForm />
+                <div className='mainContent'>
```

Comment:
Is this className doing anything? Or planning on doing something with it going forward?

Reply:
yea thats just a placeholder for future styling to be applied to anything between the header and the nav bar at the bottom of the app.